services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/AbstractPushMonitor.java

```diff
@@ -659,6 +664,22 @@ protected void onPartitionStatusChange(OfflinePushStatus offlinePushStatus) {
     checkWhetherToStartBufferReplayForHybrid(offlinePushStatus);
   }
 
+  private boolean disableReplica(PartitionAssignment partitionAssignment, Optional<String> leaderErrorInfo) {
+    String errorInfo = leaderErrorInfo.orElse("");
+    if (errorInfo.isEmpty()) {
+      return false;
+    }
+    String[] tokens = errorInfo.split("_", 2);
+    String resourceName = partitionAssignment.getTopic();
+    helixAdminClient.enablePartition(
```

Comment:
Typo? 
`enablePartiton` -> `disablePartition`

Reply:
the API is called `enablePartiton` which takes a boolean to disable/enable.